parse-args.js

```diff
@@ -62,6 +62,14 @@ function parseArgs () {
     type: 'string',
     describe: 'execute string as if inside `npm run-script`'
   })
+  .option('shell-auto-fallback', {
+    choices: ['', 'bash', 'fish', 'zsh'],
```

Comment:
As to _why I made the default `''`_... err. I don't remember. I'll have a look at the code again in its entirety to see if I can figure it out.